### ISSUE TITLE
python 2.6 sys.version_info.major failure

### ISSUE THREAD

**Issue by reconmaster** [anishathalye/dotbot#17]
Was trying to run dotbot on a server where the most recent version of python available is 2.6.  The major version check in dotbot fails to work with the following error.

```
AttributeError: 'tuple' object has no attribute 'major'
```

**Comment by anishathalye** [comment]
This was actually fixed back in 285543a (see #7). You can upgrade to the latest version by running:

```
git submodule update --remote dotbot # or other path to Dotbot submodule
```

Also, I don't think Python 2.6 supports `argparse` by default. You might have to install that [separately](https://pypi.python.org/pypi/argparse) if you haven't done so already.

**Comment by reconmaster** [comment]
The update fixed it. I didn't running install was resetting the dotbot version to the previous version I had checked in at the last commit. Thanks!
